packages/client/modules/userDashboard/components/OrgBilling/OrgPlans.tsx

```diff
@@ -233,9 +281,15 @@ const OrgPlans = () => {
               </UL>
             </Content>
             <ButtonBlock>
-              <UpgradeButton buttonStyle={plan.buttonStyle} size='medium'>
-                {plan.buttonLabel}
-              </UpgradeButton>
+              {plan.buttonStyle && (
```

Comment:
+1 I see it is defined for every plan, does it make sense to check it?

Reply:
True, thanks! Originally, that wasn't the case as we weren't going to show Plans for Enterprise users. Updating it now